common/components/Forms/AddStreetForm/index.tsx

```diff
@@ -32,6 +34,7 @@ const AddStreetForm: FC<Props> = ({ form }) => {
           placeholder="Введіть адресу"
           maxLength={256}
           className={s.formInput}
+          disabled={editable}
```

Comment:
`editable` = possible to be edited
`disabled` = NOT possible to be edited

So... `disabled={editable}` looks bad. Try to make it correct `disabled={!editable}`